src/Room/components/Player/index.tsx

```diff
@@ -23,7 +23,7 @@ const PlayerWrapper = system(
 
 const VideoPlayerIcon = system(
   {
-    is: Card,
+    as: Card,
```

Comment:
This was throwing an error in the console `Received a `object` for a string attribute 'is'`.  The `as` property does not seem to care.

Reply:
Sorry, old habit. The newer version of styled-system switched from `is` to `as`.